src/Components/ClickedComponent/ClickedComponent.tsx

```diff
@@ -19,10 +19,20 @@ let ClickedComponent = (props: ClickedComponentProps) =>{
       return newArr.join('/')
     }
     let startDate = updateDate(props.startDate)
-    let endDate = updateDate(props.endDate)
+    let endDate;
+    if(props.endDate){
```

Comment:
Updated logic to display correct message depending on if there is an end event or not

Reply:
Awesome!